pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -513,15 +613,15 @@ func TestValidateKubeSchedulerConfigurationV1beta3(t *testing.T) {
 
 	extenderDuplicateManagedResource := validConfig.DeepCopy()
 	extenderDuplicateManagedResource.Extenders[0].ManagedResources = []config.ExtenderManagedResource{
-		{Name: "foo", IgnoredByScheduler: false},
-		{Name: "foo", IgnoredByScheduler: false},
+		{Name: "example.com/foo", IgnoredByScheduler: false},
+		{Name: "example.com/foo", IgnoredByScheduler: false},
 	}
 
 	extenderDuplicateBind := validConfig.DeepCopy()
 	extenderDuplicateBind.Extenders[0].BindVerb = "foo"
 	extenderDuplicateBind.Extenders = append(extenderDuplicateBind.Extenders, config.Extender{
 		PrioritizeVerb: "prioritize",
-		BindVerb:       "bar",
+		BindVerb:       "foo",
```

Comment:
why the change here?

Reply:
Is it supposed to check duplicate name error?

Reply:
No, this one is to test duplicate `BindVerb`  indicated by the name `extenderDuplicateBind`.

Reply:
You are right. Reverted the change.

Reply:
did you forget to revert here?

Reply:
Yes, I forgot to change two versions. Reverted for all.